js/index.js

```diff
@@ -0,0 +1,34 @@
+$( document ).ready(function() {
+    console.log( "ready!" );
+
+
+    function traffic1(){
+        clearLights();
+       $("#stopLight").css("background-color", "red");
+     }
+
+    $("#stopButton").click(traffic1);
+
+    function traffic2(){
+      clearLights();
+       $("#slowLight").css("background-color", "yellow");
+     }
+
+    $("#slowButton").click(traffic2);
+
+
+    function traffic3(){
+      clearLights();
+       $("#goLight").css("background-color", "green");
+     }
+
+    $("#goButton").click(traffic3);
+
+
+    function clearLights(){
+       $("#stopLight").css("background-color", "black");
+       $("#slowLight").css("background-color", "black");
+       $("#goLight").css("background-color", "black");
+     }
```

Comment:
@sjbrat this function works great but did notice the bulbs share the class ".bulb" and you could have also used $(".bulb").css("background-color", "black"); to target all of them at once.